lib/drivers/docker/image.js

```diff
@@ -1,13 +1,11 @@
 'use strict';
 
-var cicada = require('strong-fork-cicada');
 var Container = require('./container');
-var debug = require('debug');
+var DeployReceiver = require('strong-deploy-receivers');
```

Comment:
strong-deploy-receivers also has to be added to CI and then merged so that it gets pushed to staging :-)